spec/workers/deliver_submission_worker_spec.rb

```diff
@@ -45,7 +45,9 @@
     it 'calls Submission#deliver with SQL locking' do
       # Update all records in the DB; this doesn't update the in memory Ruby
       # submission instance. This is simulating another worker running.
-      Submission.update(state: Submission::DELIVERED)
+      # rubocop:disable Rails/SkipsModelValidations
+      Submission.update_all(state: Submission::DELIVERED)
+      # rubocop:enable Rails/SkipsModelValidations
```

Comment:
Ah, so looks like `.update` now [1] only handles single objects, whereas it used to have a default `id` of `:all` [2]?

Why do we need to disable rubocop for this line?

Neither of these are blockers, but would be worth adding to the commit message for future reference.

[1] https://api.rubyonrails.org/v5.2.2/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-update
[2] https://api.rubyonrails.org/v5.2.1.1/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-update